Fix #1659, replace whitespace with dash.

pepoviola opened the pull request:
fix #1659

l0r1s commented:
@pepoviola just an after thought but maybe we could log to the user that we modified a name if we modified it ? wdyt ?

pepoviola commented:
> @pepoviola just an after thought but maybe we could log to the user that we modified a name if we modified it ? wdyt ?

Yes, sounds good. I think a warning message will be ok. Can you open an issue to track this?
Thx!
